Source/Scene/Scene.js

```diff
@@ -1606,6 +1620,7 @@ define([
         }
 
         scene._primitives.update(context, frameState, commandList);
+        scene._groundPrimitives.update(context, frameState, commandList);
```

Comment:
Do you think we should do this before `_primitives.update` so all ground primitives are drawn below all primitives if both were to have `GroundPrimitive`s?